cap-ci/scripts/deploy_k8s.tmpl

```diff
@@ -14,6 +14,23 @@ if [[ ${BACKEND} == "gke" ]]; then
   cp catapult/build${GKE_CLUSTER_NAME}/kubeclusterreference kubeconfig-pool/metadata
   echo tf-${GKE_CLUSTER_NAME}.zip > tfstate-pool/name
   cp catapult/build${GKE_CLUSTER_NAME}/tf-setup.zip tfstate-pool/metadata
+elif [[ ${BACKEND} == "aks" ]]; then
+  printf "%s" '((ssh-key-ci.pvt-key))' > $PWD/sshkey
```

Comment:
I wonder if other backends would need or benefit from the ssh key, and if so, if this block could be moved out of the `if`s

Reply:
The `gke` block exists already and it does seem to have its own way to handle `ssh` keys. We haven't added `eks` yet so can't say for sure how will it be handled there.